app/views/logs/index.html.erb

```diff
@@ -1,29 +1,30 @@
+<%= link_to "Back to search", search_logs_path, class: "govuk-back-link" %>
+
 <% if @logs.empty? %>
-  <h3>No results found for that username, please try again</h3>
+  <h3 class="govuk-heading-m">No results found for that username, please try again</h3>
```

Comment:
I feel like this language suggests our search can fail, and that being more direct will cause less people to contact us thinking it's 

I'm thinking something more like:
> No authentication attempts found.

> If this user has tried to use GovWifi, there is a problem with a local network forwarding authentication requests.

Reply:
Added to icebox.